src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  static filterAnimal() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The method `filterAnimal` is well implemented using filter, which is efficient and follows the checklist guidance to avoid nested loops for performance.